src/views/quiz-settings/quiz-settings.js

```diff
@@ -0,0 +1,114 @@
+class QuizSettings {
+    quizAbout;
+    questionsNum;
+    questionsType;
+
+    static createRadioButton(val,nameOfRadio,what,side){
```

Comment:
There are too many parameters. They are also so unintuitive.

val - replace with value;
nameOfRadio - I would change it to settingName;
what - what is "what" :P It can be removed. Pass event to the handler and use event.currentTarget.name to check which setting is clicked
side - currently, it is working fine, but this would be hard to add more buttons without rebuilding it